tests/test_io.py

```diff
@@ -85,3 +88,21 @@ def test_cast_warning(self):
             write_mrc(fname, array, 1.0)
         self.assertEqual(len(cm.output), 1)
         self.assertIn("np.float32", cm.output[0])
+
+    def test_parse_relion5_star_data(self):
+        tomogram = pathlib.Path("rec_tomo200528_107.mrc")
+        meta_data = parse_relion5_star_data(RELION5_TOMOGRAMS_STAR, tomogram)
+        self.assertEqual(len(meta_data), 5)
+        self.assertIsInstance(meta_data[0], float)
+        self.assertIsInstance(meta_data[1], list)
+        self.assertIsInstance(meta_data[2], list)
+        self.assertIsInstance(meta_data[3], list)
+        self.assertIsInstance(meta_data[3][0], dict)
+        self.assertIsInstance(meta_data[4], int)
+
+        # test name mismatch
+        tomogram = pathlib.Path("tomogram.mrc")
+        with self.assertRaises(
+            ValueError, msg="Unmatching tomograms name should raise an error."
+        ):
+            parse_relion5_star_data(RELION5_TOMOGRAMS_STAR, tomogram)
```

Comment:
please add any/all of the following tests:
```suggestion
        tomogram = pathlib.Path("tomogram.mrc")
        with self.assertRaises(
            ValueError, msg="Unmatching tomograms name should raise an error."
        ):
            parse_relion5_star_data(RELION5_TOMOGRAMS_STAR, tomogram)

        tomogram = pathlib.Path("rec_tomogram200528_1077.mrc")
        with self.assertRaises(
            ValueError, msg="Partially matching tomogram name should raise an error."
        ):
            parse_relion5_star_data(RELION5_TOMOGRAMS_STAR, tomogram)


        tomogram = pathlib.Path("rec_tomogram200528_10.mrc")
        with self.assertRaises(
            ValueError, msg="Partially matching tomogram name should raise an error."
        ):
            parse_relion5_star_data(RELION5_TOMOGRAMS_STAR, tomogram)
```